src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -16,16 +16,16 @@
     <!-- the Application Insights package -->
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.17.0" />
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="Microsoft.Win32.Registry.AccessControl" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.DirectoryServices" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.Management" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.Security.AccessControl" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="5.0.1" />
```

Comment:
I'm curios why `System.Security.Cryptography.Pkcs` was on version="5.0.1" while all neighbor items were on `6.0.0-preview.1` - is this expected or something we missed previously?

Reply:
I guess not all packages was updated in .Net Runtime for Preview.1.

Reply:
It seems we missed updating a couple of packages for preview 1. Should not be a problem this release though.